tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -283,43 +301,89 @@ def pre_checks(self):
         if self._compat_mode:
             ret += ['requires_config_enabled MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE']
 
-        if 'rsa_pss_rsae_sha256' in self._sig_algs:
+        if 'rsa_pss_rsae_sha256' in self._sig_algs + self._cert_sig_algs:
             ret.append(
                 'requires_config_enabled MBEDTLS_X509_RSASSA_PSS_SUPPORT')
         return ret
 
     def post_checks(self):
-        check_strings = ["ECDH curve: {group}".format(group=self._named_groups[0]),
-                         "server hello, chosen ciphersuite: ( {:04x} ) - {}".format(
+        check_strings = ["server hello, chosen ciphersuite: ( {:04x} ) - {}".format(
                              CIPHER_SUITE_IANA_VALUE[self._ciphers[0]],
                              self.CIPHER_SUITE[self._ciphers[0]]),
                          "Certificate Verify: Signature algorithm ( {:04x} )".format(
                              SIG_ALG_IANA_VALUE[self._sig_algs[0]]),
                          "Verifying peer X.509 certificate... ok", ]
+        for named_group in self._named_groups:
+            check_strings += ['NamedGroup: {named_group} ( {iana_value:x} )'.format(
+                                                    named_group=named_group,
+                                                    iana_value=NAMED_GROUP_IANA_VALUE[named_group])]
         return ['-c "{}"'.format(i) for i in check_strings]
 
 
 SERVER_CLASSES = {'OpenSSL': OpenSSLServ, 'GnuTLS': GnuTLSServ}
 CLIENT_CLASSES = {'mbedTLS': MbedTLSCli}
 
 
-def generate_compat_test(server=None, client=None, cipher=None, sig_alg=None, named_group=None):
+def generate_compat_test(server=None, client=None, cipher=None,
+                         cert_sig_alg=None, named_group=None):
     """
     Generate test case with `ssl-opt.sh` format.
     """
     name = 'TLS 1.3 {client[0]}->{server[0]}: {cipher},{named_group},{sig_alg}'.format(
-        client=client, server=server, cipher=cipher, sig_alg=sig_alg, named_group=named_group)
-    server_object = SERVER_CLASSES[server](cipher, sig_alg, named_group)
-    client_object = CLIENT_CLASSES[client](cipher, sig_alg, named_group)
+        client=client, server=server, cipher=cipher, sig_alg=cert_sig_alg, named_group=named_group)
+
+    server_object = SERVER_CLASSES[server](ciphersuite=cipher,
+                                           named_group=named_group,
+                                           cert_sig_alg=cert_sig_alg)
```

Comment:
```suggestion
                                           signature_algorithm=cert_sig_alg,
                                           cert_sig_alg=cert_sig_alg)
```
as for the client object.